packages/ws/src/ws/WebSocketManager.ts

```diff
@@ -127,6 +138,10 @@ export interface OptionalWebSocketManagerOptions {
 	 * How long to wait for a shard's READY packet before giving up
 	 */
 	readyTimeout: number | null;
+	/**
+	 * The REST instance to use for fetching gateway information
+	 */
+	rest?: REST;
```

Comment:
Now we can deprecate (or remove) this 😄

Reply:
I dont want this PR to become a major. We can re-deprecate it >.>